CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ The Azure MCP Server updates automatically by default whenever a new release com
 
 - Added support for providing the `--content-type` and `--tags` properties to the `azmcp-appconfig-kv-set` command. [[#459](https://github.com/Azure/azure-mcp/pull/459)]
 - Added `filter-path` and `recursive` capabilities to `azmcp-storage-datalake-file-system-list-paths`. [[#770](https://github.com/Azure/azure-mcp/issues/770)]
-
+- Added support for listing files and directories in Azure File Shares via the `azmcp-storage-share-file-list` command. This command recursively lists all items in a specified file share directory with metadata including size, last modified date, and content type. [[#793](https://github.com/Azure/azure-mcp/pull/793)]
```

Comment:
Maybe we should confirm the tool names with the storage team. Personally, I feel the names are confusing.

Reply:
I'm good as-is

Reply:
Sounds goods on just going forward now with what is in the PR to unblock work.

I do agree with both you @xiangyan99 and @alzimmermsft there's probably a better naming scheme from an MCP tool perspective than trying to follow exactly the naming scheme that is in the SDKs today, but would like to take the time to think through it a bit more.

Reply:
According to our architect, renaming tools are not considered breaking changes provided we set clear expectations with our customers.

Reply:
> According to our architect, renaming tools are not considered breaking changes provided we set clear expectations with our customers.

@xiangyan99 it IS a breaking change, and WILL be listed as one in the CHANGELOG.md, however, this is pre-1.0 / pre-release and this is fine.